sources/Bundle/FeatureTogglesExtension.php

```diff
@@ -31,12 +31,12 @@ public function load(array $configs, ContainerBuilder $container)
         $this->defineDoctrineEventSubscriber($container);
     }
 
-    public function getConfiguration(array $config, ContainerBuilder $container)
+    public function getConfiguration(array $config, ContainerBuilder $container): FeatureTogglesConfiguration
     {
         return new FeatureTogglesConfiguration();
     }
 
-    private function defineFeatureRegistry(array $declaredFeatures, ContainerBuilder $container)
+    private function defineFeatureRegistry(array $declaredFeatures, ContainerBuilder $container): void
```

Comment:
I do not see how adding return typehints on private methods like here can fix any deprecation.

Can you explain where the deprecation is reported?